src/quo/components/wallet/keypair/view.cljs

```diff
@@ -70,10 +70,11 @@
       {:size  :paragraph-2
        :style (style/subtitle blur? theme)}
       address]
-     [text/text
-      {:size  :paragraph-2
-       :style (merge (style/subtitle blur? theme) {:bottom (if platform/ios? 2 -2)})}
-      " ∙ "]
+     (when (= type :default-keypair)
+       [text/text
+        {:size  :paragraph-2
+         :style (merge (style/subtitle blur? theme) {:bottom (if platform/ios? 2 -2)})}
```

Comment:
why is there merge being done outside? 
imo it's better to create another style def and move this logic into the style file

Reply:
ah it's already that way, feel free to ignore if you want so 👍

Reply:
Fixed! 👍